[KOGITO-5300] Updating documentation of addons modules (#1463)
kiegroup/kogito-runtimes: addons/common/monitoring/README.md

# Kogito Monitoring Add-ons

> The Kogito Monitoring Add-ons add monitoring capabilities to Kogito projects. See 

Reviewer: ```suggestion
The Kogito Monitoring Add-ons adds monitoring capabilities to Kogito projects. See 
```

> These are the addons for adding monitoring capabilities to Kogito projects. Please check

Reviewer 2: ```suggestion
The Kogito Monitoring Addons add monitoring capabilities to Kogito projects. See 
```
